### PR TITLE
Update example cfg

### LINKED ISSUE #712: [Bug]: Updates needed for the example cfg
### What happened?

1. fix: streamflow enso sets in e3sm_diags are not enabled in the config
2. update: to use coupled group's config for mpas-analysis and global time-series plot
3. (nice to have) include headers for pm-cpu and compy for easier reference.

### What machine were you running on?

Chrysalis

### Environment

main branch

### What command did you run?

```bash
na
```

### Copy your cfg file

```bash
zppy/example/post.v3.LR.historical.zppy_v3.cfg
```

### What jobs are failing?

```bash

```

### What stack trace are you encountering?

```bash

```

### PR DESCRIPTION
**Pull request by chengzhuzhang**
## Summary

    1.  enable streamflow enso sets in e3sm_diags  in the config
    2. update to use coupled group's config for mpas-analysis and global time-series plot: use 1850 as baseline. Including comments for clarifying parameters regarding years
    3. comment out `environment_commands` that activate LCRC specific e3sm_unified path, to make the .cfg more portable on other supported machines.

Issue resolution:
- Closes #712

Select one: This pull request is...
- [ ] a bug fix: increment the patch version
- [x] a small improvement: increment the minor version
- [ ] a new feature: increment the minor version
- [ ] an incompatible (non-backwards compatible) API change: increment the major version

Please fill out either the "Small Change" or "Big Change" section (the latter includes the numbered subsections), and delete the other.

## Small Change

- [ ] To merge, I will use "Squash and merge". That is, this change should be a single commit.
- [ ] Logic: I have visually inspected the entire pull request myself.
- [ ] Pre-commit checks: All the pre-commits checks have passed.

## Big Change

- [ ] To merge, I will use "Create a merge commit". That is, this change is large enough to require multiple units of work (i.e., it should be multiple commits).

### 1. Does this do what we want it to do?

Required:
- [ ] Product Management: I have confirmed with the stakeholders that the objectives above are correct and complete.
- [ ] Testing: I have added or modified at least one "min-case" configuration file to test this change. Every objective above is represented in at least one cfg.
- [ ] Testing: I have considered likely and/or severe edge cases and have included them in testing.

If applicable:
- [ ] Testing: this pull request introduces an important feature or bug fix that we _must_ test often. I have updated the weekly-test configuration files, not just a "min-case" one.
- [ ] Testing: this pull request adds at least one new possible parameter to the cfg. I have tested using this parameter with and without any other parameter that may interact with it.

### 2. Are the implementation details accurate & efficient?

Required:
- [ ] Logic: I have visually inspected the entire pull request myself.
- [ ] Logic: I have left GitHub comments highlighting important pieces of code logic. I have had these code blocks reviewed by at least one other team member.

If applicable:
- [ ] Dependencies: This pull request introduces a new dependency. I have discussed this requirement with at least one other team member. The dependency is noted in `zppy/conda`, not just an `import` statement.

### 3. Is this well documented?

Required:
- [ ] Documentation: by looking at the docs, a new user could easily understand the functionality introduced by this pull request.

### 4. Is this code clean?

Required:
- [ ] Readability: The code is as simple as possible and well-commented, such that a new team member could understand what's happening.
- [ ] Pre-commit checks: All the pre-commits checks have passed.

If applicable:
- [ ] Software architecture: I have discussed relevant trade-offs in design decisions with at least one other team member. It is unlikely that this pull request will increase tech debt.

### CODE REVIEW

**Comment by chengzhuzhang** [comment]
After changing ts_years to 1850 as baseline, the global_time_series job exceed wall time limit (10hr). This is the only error when I tested this new cfg and likely related to the performance issue discovered earlier with zi-global-time-series. We will have to wait to merge this PR until the performance issue is addressed.

**Comment by forsyth2** [comment]
Ok that makes sense; it looks like we have a chain of PRs to get through for this:
1. https://github.com/E3SM-Project/zppy-interfaces/issues/23
2. https://github.com/E3SM-Project/zppy-interfaces/pull/22 && https://github.com/E3SM-Project/zppy-interfaces/pull/19
3. This PR

File: examples/post.v3.LR.historical.zppy_v3.cfg [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+# Users should only enable the tasks and time periods they need based on their specific
+# analysis requirements to reduce computational load and queue time.
 
 [default]
 # Change these two paths to include your username instead
 # You can also change these to match your preferred directory structure
-output = /lcrc/group/e3sm/ac.forsyth2/zppy_example_v3/v3.LR.historical_0051
-www = /lcrc/group/e3sm/public_html/diagnostic_output/ac.forsyth2/zppy_example_v3
+output = /lcrc/group/e3sm/ac.zhang40/zppy_example_v3.1/v3.LR.historical_0051
+www = /lcrc/group/e3sm/public_html/diagnostic_output/zppy_example/v3.1
```
Comment (forsyth2) [inline examples/post.v3.LR.historical.zppy_v3.cfg]: I might recommend making this `v3.1.0` to match the version exactly.

**Review by forsyth2 — APPROVED** [review]
@chengzhuzhang Thanks, this looks good to me.

**Comment by chengzhuzhang** [comment]
@forsyth2 thanks for a review! I have addressed your comment, and will merge when checks are completed.
